pkg/operator/ceph/file/filesystem.go

```diff
@@ -175,6 +189,9 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 					fmt.Sprintf(". %v", err),
 			)
 		}
+		if err := SetPoolSize(context, clusterName, f.Name, f.dataPools, f.metadataPool); err != nil {
+			return errors.Wrapf(err, "failed to set pool size to %d", f.metadataPool.ReplicatedConfig.Size)
```

Comment:
remove the details of the size from this message since the metadata settings could be different from the data pools. The details should be in the wrapped err.

Reply:
Wrapped error?. I edited it and made changes like this
`return errors.Wrapf(err, "failed to set pools size")`